lib/systems/examples.nix

```diff
@@ -90,6 +90,11 @@ rec {
     config = "mipsel-unknown-linux-gnu";
   } // platforms.fuloong2f_n32;
 
+  loongnix = {
```

Comment:
Where does this name come from? I'd expect something more like "loongarch64-linux".

Reply:
Sorry, It was because my mistake, I have read the docs about how to name it, but when I write it I forgot it (now even I couldn't find it).
The `loongnix` is actually the operating system name which run in the machine that I got.